library-model-loader/build.gradle

```diff
@@ -35,3 +35,15 @@ dependencies {
     annotationProcessor deps.build.nullaway
     compileOnly deps.build.guava
 }
+
+import net.ltgt.gradle.errorprone.CheckSeverity
+
+tasks.withType(JavaCompile) {
+    // remove the if condition if you want to run NullAway on test code
+    if (!name.toLowerCase().contains("test")) {
+        options.errorprone {
+            check("NullAway", CheckSeverity.ERROR)
+            option("NullAway:AnnotatedPackages", "edu")
```

Comment:
A bit aggressive. This should be at most `edu.ucr`, and arguably `edu.ucr.cs.riple`, I think (cc: @msridhar ), otherwise you are relying on no dependencies ever being `edu.*`

Reply:
Edit: Note that what I am suggesting here would require all test examples to be inside a package like `edu.ucr.cs.riple[.scanner.test]`, but that is probably best practice anyways.

Reply:
@nimakarimipour No action here? Why?

Reply:
Sorry for missing this comment and thank you for noticing this. I actually wrote "edu" just because there wasn't significant code in this library and thought it only affects the code available in `src` directory which is not true.

I also inspired from `NullAway Library Models` sample. Now that I have a much better understanding of how these works and the correct configurations, I see that this was only there as the whole mission of enrolling NullAway into NullAway and library models loader does not need `NullAway` to be on it's annotation processor. Removed it. 01a15119f5e34bbbc475210b92e9ed390f501ba6